src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -173,6 +196,14 @@ export default {
 			type: String,
 			default: null,
 		},
+
+		/**
+		 * Enable content focus trap
+		 */
+		focusTrap: {
+			type: Boolean,
+			default: true,
+		},
```

Comment:
We just made a breaking change in v9 to get rid of boolean props with default true value, as they conflict with the HTML template style.

Let's call it `disableFocusTrap` or `noFocusTrap`.

Reply:
Or we can check if there are any focusable elements and enable/disable it automatically.

Reply:
Ref: https://github.com/nextcloud-libraries/nextcloud-vue/pull/6452